averbis/core/_rest_client.py

```diff
@@ -775,9 +785,19 @@ def export_text_analysis(self, annotation_types: str = None) -> dict:
                 "Text analysis export is not supported for platform version 5.x, it is only supported from 6.x onwards."
             )
 
+        if (page is not None and page_size is None) or (page is not None and page_size is None):
```

Comment:
How about using a default page size (e.g. 100) and allow the user to specify only the page parameter?

Reply:
Thanks for the feedback. I changed the logic in the latest commit according to that suggest.